ines/ines.js

```diff
@@ -0,0 +1,21 @@
+let apples = 20;
+let oranges = 30;
+
+console.log("Line 4 is:", apples === oranges ? true : false);
+console.log("Line 5 is:", apples !== oranges ? true : false);
+console.log("Line 6 is:", apples < oranges ? true : false);
```

Comment:
you missed exercise 4! Check whether apples is less than or equal to oranges.
here is the solution!
console.log(apples <= oranges);